messages/protos/messages.proto

```diff
@@ -218,6 +218,18 @@ message ErrorMessage {
   string message = 1;
 }
 
+message ErrorV2Message {
+  string type = 1;
+  string code = 2;
+  string title = 3;
+  string subtitle = 4;
+  string details = 5;
+  int32 timeToRetry = 6;
+  bool canRetryManual = 7;
+  string urlToRedirect = 8;
+  string buttonTitle = 9;
+}
```

Comment:
We will have to check where in the code we use the old `ErrorMessage` and if we can replace this with the new `ErrorV2Message` (which is way more violent because it displays an error screen instead of simply adding a small error log in the console)